python/vegafusion/tests/altair_mocks/line/with_generator/mock.py

```diff
@@ -2,7 +2,7 @@
 
 import altair as alt
 
-source = alt.sequence(start=0, stop=12.7, step=0.1, as_='x')
+source = alt.sequence(start=0, stop=12.71, step=0.1, as_='x')
```

Comment:
The stop value of a sequence is exclusive, so whether 12.7 is included here is determined by floating point error propagation. Increased a little so that 12.7 is clearly to be included.